godot-core/src/builtin/collections/array.rs

```diff
@@ -1334,7 +1334,7 @@ impl<T: ArrayElement> GodotType for Array<T> {
         // Typed arrays use type hint.
         PropertyHintInfo {
             hint: crate::global::PropertyHint::ARRAY_TYPE,
-            hint_string: GString::from(element_godot_type_name::<T>()),
+            hint_string: GString::from(&element_godot_type_name::<T>()),
```

Comment:
Aside: I wonder if `element_godot_type_name` and thus `godot_type_name` could/should return `&str` instead. Probably someone thought about this before and the answer is "no" :sweat_smile: